### ISSUE TITLE
Debugging Console opens, Installing Mod from ZIP still doesn't work

### ISSUE THREAD

**Issue by DismalAmoeba** [ModulousSmash/ModulousMM#7]
Nothing happens. Debugging Console remains blank. Perhaps my mod isn't set up correctly, but I have no idea what's wrong.

Test folder is set up with Modulous and Private folder in it, with SFX folder inside that in the correct pf folder.

Folder supposed to be put onto root is also a folder named Private, which has the correct folders and BrawlEX inside.  

In my LUA file. 
SDCard.copy_from_mod_to_sd_card("private/", "/")
CharacterManager.add_character_from_mod_root("ft_younglink.rel", "Cosmetic43.dat", "Slot43.dat", "Fighter43.dat", "CSSSlot43.dat", "younglink" , "Young Link")

**Comment by EIREXE** [comment]
I forgot to redirect stuff to the console, update to the latest version I just pushed and after that if you get any errors on the console you can send in the mod zip files so I can take a look.

**Comment by DismalAmoeba** [comment]
Same updater bug happens as before, and fixed the same way.

**Comment by EIREXE** [comment]
I fixed the updating bug.

Everyone will need to update manually.

**Comment by DismalAmoeba** [comment]
I was right, it won't let you create a file when it already exists.

**Comment by EIREXE** [comment]
Actually the add_character function already copies the character files, can you send in the ZIP so I can check it out and tell you were is the issue?

**Comment by DismalAmoeba** [comment]
Yeah, it isn't working out. Here is the zip.

https://www.dropbox.com/s/etd0jcctbarshzb/younglink.zip?dl=0

**Comment by EIREXE** [comment]
One thing you might want to know is that the slot is automatically changed, so at the moment you can't replace characters, add_characters adds a character at the next available slot.

https://mega.co.nz/#!G55gjbwR!rpgQWJTfincidRsYeIS1xfBm29FFp4hc_49d96gNTKE
Include file key

**Comment by DismalAmoeba** [comment]
Error: BrawlEX + CSS Expansion Mod required. Even on a build of Brawl Minus. Tried it twice.

![capture](https://cloud.githubusercontent.com/assets/13057910/8370794/c4d920ac-1b92-11e5-9d24-c0f409dea721.PNG)

**Comment by EIREXE** [comment]
IIRC brawl minus does not have CSS expansion.

To avoid the error on your console close the manager and open it again.

**Comment by DismalAmoeba** [comment]
I'm 100% sure that BrawlEX has Roy and Pichu as clone characters and I checked the RSBE01- it does.

**Comment by EIREXE** [comment]
Yes it does, using the old method of editing the CSS from code that can be done with BrawlEX only, what the modulous mod manager does is edit the CSSRoster.dat file that is used in BrawlEX CSS expansion.

**Comment by DismalAmoeba** [comment]
Well, crap. How do I add this?

**Comment by EIREXE** [comment]
I don't think you can with brawl minus as it overwrites the CSS, you could try installing it on top, and then pray for stuff like fox's airdodge to not break.

you can then add Roy and Pichu to the CSSRoster.dat.

**Comment by DismalAmoeba** [comment]
How do I set up a SD card to work with this? I tried with a random Test folder and placed the BrawlEX stuff in the corresponding locations and the CSSRoster.dat file in the BrawlEX folder. 

![capture2](https://cloud.githubusercontent.com/assets/13057910/8382532/dc67b98e-1bf8-11e5-888c-c1473a43dcd0.PNG)

This is after the CSSRoster.dat is added.

**Comment by EIREXE** [comment]
This is how the structure is on my SD.

└─private
    └─wii
        └─app
            └─RSBE
                └─pf
                    ├─BrawlEx
                    │  ├─CosmeticConfig
                    │  ├─CSSSlotConfig
                    │  ├─FighterConfig
                    │  └─SlotConfig
                    ├─fighter
                    │  ├─oldlink
                    │  └─younglink
                    ├─Menu2
                    ├─Module
                    └─System
The private folder is in the root.

Also you can't name your module ft_link, since it already exists.
